src/view/editor/parts/editor-composer.tsx

```diff
@@ -98,39 +104,36 @@ export const EditorComposer = ({ editorId, callbacks }: ComposerProps): ReactEle
 
 	const onRichTextChange = useCallback(
 		(e) => {
+			setPlainTextValue(e[1]);
 			throttleInput(e);
 		},
 		[throttleInput]
 	);
 
 	const onPlainTextChange = useCallback(
 		(e) => {
-			setPlainTextValue(e[0]);
-			throttleInput(e);
+			setPlainTextValue(e.target.value);
+			throttleInput([e.target.value, e.target.value]);
```

Comment:
don't we need to add atleast a <p> tag in case of richText ?

Reply:
nope, we just need to enable richText, user will decide how to manipulate the text